packages/playground/src/weblets/tf_contracts_list.vue

```diff
@@ -148,7 +189,23 @@ function getStateColor(state: ContractStates): string {
   }
 }
 
+async function contractLockDetails(contractId: number) {
+  contractStateDialog.value = true;
+  loading.value = true;
+  const grid = await getGrid(profileManager.profile!);
+  await grid?.contracts
+    .contractLock({ id: contractId })
+    .then((data: ContractLock) => {
+      contractLocked.value = data;
+    })
+    .catch(err => {
+      layout.value.setStatus("failed", normalizeError(err, `Failed to fetch the contract lock details.`));
```

Comment:
add the contract Id to the error message please

Reply:
Note: 
Using
```ts
layout.value.setStatus("failed", normalizeError(err, `Failed to fetch the contract lock details.`));
```
won't close the dialog also it will show the error in the entire contracts list view so it will confuse the user

Reply:
@MohamedElmdary 
in case we closed the dialog when there is an error, will the error still appear?